cmd/calendarsync/main.go

```diff
@@ -81,11 +80,23 @@ func main() {
 			return nil
 		},
 		Action: Run,
+		Commands: []*cli.Command{
```

Comment:
I'm wondering whether a flag (`--version` instead of `version`) would be more consistent with the remainder of the CLI? Right now, the only other command is the `help` command which seems to be autogenerated.

Reply:
I thought about this as well. It seems there isn't _the way_ to go here. I just started a poll inside our dev community at inovex to gather some impressions about preferences.

Reply:
done.